sql/src/main/java/org/apache/druid/sql/calcite/planner/ValidationResult.java

```diff
@@ -24,11 +24,11 @@
 
 import java.util.Set;
 
-public class ResourceResult
+public class ValidationResult
```

Comment:
Please add some javadoc since now it's not intuitive what `resources` in `ValidationResult` mean.